csharp/src/Google.Protobuf/Reflection/DescriptorPool.cs

```diff
@@ -181,7 +176,7 @@ internal void AddSymbol(IDescriptor descriptor)
             descriptorsByName[fullName] = descriptor;
         }
 
-        private static readonly Regex ValidationRegex = new Regex("^[_A-Za-z][_A-Za-z0-9]*$",
+        private static readonly Regex ValidationRegex = new("^[_A-Za-z][_A-Za-z0-9]*$",
```

Comment:
It's probably reasonable to unwrap the next line into this one now.

Reply:
Missed that conversion, restored as single-line